packages/logger/index.js

```diff
@@ -22,6 +22,7 @@ if (process.platform !== 'darwin') {
 }
 
 var logLevels = {
+  'none': 0,
```

Comment:
What's the none for?

Reply:
`none` is a level for disable log upload